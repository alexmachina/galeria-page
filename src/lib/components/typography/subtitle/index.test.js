import React from "react";
import Subtitle from "./";
import renderer from "react-test-renderer";

describe("Components", () => {
  describe("Typography", () => {
    describe("Subtitle", () => {
      it("Renders correctly", () => {
        const tree = renderer
          .create(<Subtitle>Hello, There!</Subtitle>)
          .toJSON();

        expect(tree).toMatchSnapshot();
      });
    });
  });
});
