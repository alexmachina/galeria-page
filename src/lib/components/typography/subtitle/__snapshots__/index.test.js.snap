// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Typography Subtitle Renders correctly 1`] = `
<span
  className="root-0-2-1"
>
  Hello, There!
</span>
`;
